### PR TITLE
Update the Full Node Documentation and remove unecessary file

### PR DESCRIPTION
**Pull request by itsyaasir**

### CODE REVIEW

File: docs/full-node-api/overview.md [hunk 1]
```diff
@@ -2,37 +2,86 @@
...
-Bitfinity currently supports a read-only full-node implementation to access the data on the EVM. Our Full node is a fork of Erigon that enables a user to download historical blocks, executes all transactions and validates the state roots. In addition, the EVM is capable of signing blocks, and the block-importer can validate the signatures to ensure the blcoks were indeed generated by the EVM.  
+### What is a Full Node?
 
-We will shortly be upgrading the full node, to enable broadcasting transactions. 
+A full node plays a pivotal role in blockchain networks by validating transactions and blocks in accordance with the network's consensus rules. It contributes to the decentralization and security of the network, offering users a private and trustless interface to the blockchain.
 
-## Build and Run a Full Node
+## Bitfinity Full Node Overview
 
-To sync with the node you will need to build the code by following the instructions in https://github.com/bitfinity-network/erigon/blob/evmc_importer/BITFINITY.README.md on the evmc_importer branch.
```
Comment (GraDKh) [inline docs/full-node-api/overview.md:15]: Actually block-importer only re-executes a block. Another important part is rpcdaemon that processes JSON RPC requests. Both together provide similar to full node functionality

File: docs/full-node-api/overview.md [hunk 2]
```diff
@@ -2,37 +2,86 @@
...
+```sh
+git clone -b evmc_importer https://github.com/bitfinity-network/erigon.git
 ```
 
-The JSON_RPC_URL must have all historical blocks.  You can retrieve all historical blocks from the following URLs: 
+For detailed build instructions, refer to the [Bitfinity Readme File](https://github.com/bitfinity-network/erigon/blob/evmc_importer/BITFINITY.README.md).
+
+### 2. Run the Full Node
+
+The block-importer operates through two processes: one for downloading and validating blocks, and another to set up an RPC URL for the full node.
```
Comment (GraDKh) [inline docs/full-node-api/overview.md:45]: I'd rather put all this information in `https://github.com/bitfinity-network/erigon/blob/evmc_importer/BITFINITY.README.md` and just reference here

File: docs/getting-started/block-explorer.md [hunk 3]
```diff
@@ -7,17 +7,20 @@ sidebar_position: 7
...
 
 ## Local Deployment
 
-You can launch a standalone version of the block explorer for access to Bitfinity's blocks and transactions. 
+You can launch a standalone version of the block explorer for access to Bitfinity's blocks and transactions.
 
-To do this, you will need to run a forked Erigon Node against the Bitfinity EVM. And once this has been stood up, you can run Otterscan or Bitfinity-Otterscan against this node. 
+To do this, you will need to run a forked Erigon Node against the Bitfinity EVM. And once this has been stood up, you can run Otterscan or Bitfinity-Otterscan against this node.
 
 [bitfinity-eragon](https://github.com/bitfinity-network/erigon)
```
Comment (GraDKh) [inline docs/getting-started/block-explorer.md:25]: Typo: `eragon` -> `erigon`

**Review by GraDKh — APPROVED** [review]

File: docs/full-node-api/overview.md (hunk 1 above)
Comment (itsyaasir) [reply docs/full-node-api/overview.md:15]: This part has been removed

**Pull request closed without merging**